app/schemas/subscriptions/tome.coffee

```diff
@@ -89,7 +89,7 @@ module.exports =
   'tome:problems-updated': c.object {title: 'Problems Updated', description: 'Published when problems have been updated', required: ['spell', 'problems', 'isCast']},
     spell: {type: 'object'}
     problems: {type: 'array'}
-    isCast: {type: 'boolean'}
+    isCast: {type: 'boolean'} # Whether the code has been Run yet. Sometimes determines if error displays as just annotation or as full banner.
```

Comment:
You can put this in the `description` in the schema instead of just in a comment